test/tests/pkgmaint-scripts.bats

```diff
@@ -11,7 +11,7 @@ load ../util/util
     makedeb -d
 
     ar xf testpkg_1.0.0-1_all.deb
-    mapfile -t files < <(tar tf control.tar.gz | sort -V)
+    mapfile -t files < <(tar tf control.tar.zst | sort -V)
     mapfile -t expected_files < <(printf '%s\n' './control' './preinst' | sort -V)
 
     [[ "${#files[@]}" == "${#expected_files[@]}" ]]
```

Comment:
Can we rename this to `archive.bats` or something like that since we're dealing with how makedeb is creating archives?